packages/sdks/src/functions/get-content/generate-content-url.ts

```diff
@@ -15,14 +15,19 @@ export const generateContentUrl = (options: GetContentOptions): URL => {
     apiKey,
     includeRefs = true,
     locale,
+    apiVersion = 'v2',
   } = options;
 
   if (!apiKey) {
     throw new Error('Missing API key');
   }
 
+  if (!['v1', 'v2', 'v3'].includes(apiVersion)) {
+    throw new Error('Invalid apiVersion value');
```

Comment:
I would also improve this message to be explicit about what is supported

```suggestion
    throw new Error(`Invalid apiVersion: expected 'v2' or 'v3', received '${apiVersion}'`);
```